ocw/lib/gce.py

```diff
@@ -187,9 +187,14 @@ def cleanup_images(self) -> None:
         self.log_dbg(f"{len(images)} images found")
         for image in images:
             if self.is_outdated(parse(image["creationTimestamp"]).astimezone(timezone.utc)):
-                self._delete_resource(
-                    self.compute_client().images, image["name"], project=self.project, image=image["name"]
-                )
+                labels = image.get('labels', [])
+                pcw_ignore_tag = 'pcw_ignore' in labels
+                if pcw_ignore_tag:
+                    self.log_dbg(f"Ignoring {image['name']} due to 'pcw_ignore' label set to '1'")
```

Comment:
```suggestion
                    self.log_dbg(f"Ignoring {image['name']} due to 'pcw_ignore' in labels")
```

all over the code including this patch the only requirement that flag **exists**  PCW never put any requirements about value of the flag